tests/containers/podman_quadlet.pm

```diff
@@ -136,12 +137,25 @@ sub run {
     systemctl("disable --now $unit_name.service");
     systemctl("stop $unit_name-volume.service");
     systemctl("stop $unit_name-network.service");
-    systemctl("stop $unit_name-build.service");
+    systemctl("stop $unit_name-build.service", ignore_failure => 1);
```

Comment:
Why is this needed?

Reply:
It's either that or

```perl
if (check_min_runtime_version('5.2.0')) {
    systemctl("stop $unit_name-build.service");
}
```

Reply:
Suggestion: Introduce `my $has_build = check_min_runtime_version('5.2.0')` at the top and then something like

```
systemctl("stop $unit_name-build.service") if($has_build);
```

Might also help to avoid running this check multiple times.

Reply:
Ah good idea, fixed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20751/commits/f4b18942cae36b2a523995274024bbc3e33bb79d